org.gnome.Music.json

```diff
@@ -55,7 +52,7 @@
             "buildsystem": "meson",
             "subdir": "subprojects/gst-python",
             "config-opts": [
-                "-Dpygi-overrides-dir=/app/lib/python3.11/site-packages/gi/overrides/"
+                "-Dpygi-overrides-dir=/app/lib/python3.12/site-packages/gi/overrides/"
```

Comment:
This is wrong!! GNOME 46 still uses 3.11.

Reply:
Nice catch.

It is possible to run Music as-is without gst-python (as we noticed). I believe it mainly provides some convenience pythonic API additions. Possible we have (had?) some code-paths that utilize this, otherwise it probably should be dropped as dep.

I'll open an issue in our bugtracker to investigate.

Reply:
For the record, I saw some warnings when starting the app when this was using 3.11. I had the misconception that the runtime included 3.12 then made this change and the warning disappeared. 

I will look at bit more at it when I get home, but I suspect that just bumping to match the runtime (1.22.10) is good enough.

Reply:
The error in question is
```
$ flatpak run org.gnome.Music

** (gst-plugin-scanner:7): CRITICAL **: 18:17:24.739: Couldn't g_module_open libpython. Reason: /usr/lib/x86_64-linux-gnu/libpython3.11.so: cannot open shared object file: No such file or directory
```
But it only appears the *first* time you open the app (delete `~/.var/app/org.gnome.Music` to try it) and it is reproducible both in stable and this PR.

I fixed the wrong python version and took the opportunity to update the shared-modules submodule.

Reply:
This seems to be the issue: https://github.com/flatpak/flatpak/issues/537 .

It's not harmful to Music's use case.